components/list_modal.jsx

```diff
@@ -153,6 +158,9 @@ export default class ListModal extends React.PureComponent {
     }
 
     render() {
+        if (this.props.show === false) {
```

Comment:
Maybe `if (!this.props.show) {` instead?

Reply:
I have added an explicit default `show` prop value so that it can be written as you have proposed.